wallet-new/src/Cardano/Wallet/Orphans/Arbitrary.hs

```diff
@@ -28,5 +29,20 @@ instance Arbitrary CPaperVendWalletRedeem
 instance Arbitrary CInitialized
 instance Arbitrary NewBatchPayment
 
+instance Arbitrary CCoin where
+    arbitrary = CCoin <$> arbitrary
+
+instance Arbitrary CUpdateInfo where
```

Comment:
Weird, why was this `Arbitrary` instance needed as part of this PR?

Reply:
It was needed to fix Swagger issues - see PR description above 
> Note 2: This PR includes a fix for Swaggers scheme of WalletStateSnapshot (part of CSL-2094) :